src/pages/workspace/taxes/ValuePage.tsx

```diff
@@ -87,9 +87,12 @@ function ValuePage({
                             <InputWrapper
                                 InputComponent={AmountForm}
                                 inputID={INPUT_IDS.VALUE}
-                                defaultValue={value}
-                                onInputChange={setValue}
+                                defaultValue={defaultValue}
                                 hideCurrencySymbol
+                                // The default currency uses 2 decimal places, so we substract it
+                                extraDecimals={CONST.MAX_TAX_RATE_DECIMAL_PLACES - 2}
+                                // We increase the amount max length. We have to add 2 places for one digit and comma.
```

Comment:
is `one digit and comma` enough? Does it mean that user can't enter eg. `99.(16 digits)`?

Reply:
You're right, might be an edge case here to handle. I will check it when I get a confirmation that we need that many decimal places.

Reply:
We definitely need 2 digits before the comma, since users can enter something like 10.75% as a tax rate

Reply:
Okay, I've changed it to 4 decimal places, and it works for both cases: `11.1111` and `1.1111`.  The user also cannot set tax value to 5 decimal places (`1.11111`).

I'm marking it as ready for review.

Reply:
Thank you!